apis/duck/cached.go

```diff
@@ -38,35 +38,53 @@ var _ InformerFactory = (*CachedInformerFactory)(nil)
 // Get implements InformerFactory.
 func (cif *CachedInformerFactory) Get(gvr schema.GroupVersionResource) (cache.SharedIndexInformer, cache.GenericLister, error) {
 	cif.m.Lock()
+
+	var ic *informerCache
```

Comment:
remove this variable.

Reply:
[Addressed](https://github.com/knative/pkg/compare/969fb7aca97d3cd03fb7456e9b5e19ca9a9bc9ea..27fcb36f1cec5f30c95682459ef3e3314183d2df)